smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/PaginationGenerator.java

```diff
@@ -121,6 +121,11 @@ static void generateServicePaginationInterfaces(
         });
     }
 
+    public static String destructurePath(String path) {
```

Comment:
nit: this class is package-private, so this method being public is ignored. Ideally this should be private unless you're trying to share this functionality with other classes for some reason.

Reply:
Yep good point. Will do